src/Reservoir.jl

```diff
@@ -63,30 +92,27 @@ function initreservoir(m::Model, name=nothing)
         reservoir = addcomponent(m, Reservoir, name)
     end
 
-    reservoir[:inflows] = zeros(m.indices_counts[:reservoirs],m.indices_counts[:time]);
-    reservoir[:captures] = zeros(m.indices_counts[:reservoirs],m.indices_counts[:time]);
+    reservoir[:captures] = cached_fallback("extraction/captures$suffix", () -> zeros(m.indices_counts[:reservoirs], m.indices_counts[:time]));
 
     if config["netset"] == "three"
         reservoir[:storagecapacitymax] = 8.2*ones(numreservoirs)
         reservoir[:storagecapacitymin] = 0.5*ones(numreservoirs)
         reservoir[:storage0] = 1.3*ones(numreservoirs)
         reservoir[:evaporation] = 0.01*ones(numreservoirs, numsteps)
     else
-	   if config["rescap"] == "zero"
-             reservoir[:storagecapacitymax] = zeros(m.indices_counts[:reservoirs]);
+	if config["rescap"] == "zero"
+           reservoir[:storagecapacitymax] = zeros(m.indices_counts[:reservoirs]);
      	   reservoir[:storagecapacitymin] = zeros(m.indices_counts[:reservoirs]);
      	   reservoir[:storage0] = zeros(m.indices_counts[:reservoirs]);
      	   reservoir[:evaporation] = zeros(m.indices_counts[:reservoirs],m.indices_counts[:time]);
-     	   reservoir[:captures] = cached_fallback("extraction/captures", () -> zeros(m.indices_counts[:reservoirs], m.indices_counts[:time]));
         else
-	        rcmax = convert(Vector{Float64}, reservoirdata[:MAXCAP])
+	   rcmax = convert(Vector{Float64}, reservoirdata[:MAXCAP])
      	   rcmax = rcmax*1233.48
      	   reservoir[:storagecapacitymax] = rcmax;
-     	   reservoir[:storagecapacitymin] = 0*0.1*rcmax;
-     	   reservoir[:storage0] = 0*(rcmax-0.1*rcmax)/2; #initial storate value: (max-min)/2
-     	   reservoir[:evaporation] = 0.01*ones(m.indices_counts[:reservoirs],m.indices_counts[:time]);
-     	   reservoir[:captures] = cached_fallback("extraction/captures", () -> zeros(m.indices_counts[:reservoirs], m.indices_counts[:time]));
-	   end
+	   reservoir[:storagecapacitymin] = zeros(numreservoirs);
+	   reservoir[:storage0] = rcmax*0.#zeros(numreservoirs); 
```

Comment:
Yes. I believe your initial idea was to see the storage not as an absolute value but the amount that we can actually use (thus we do not have to worry about minimum amount necessary to work).

I think this makes sense, in particular in two set-ups:
1. To explore the role of storage to mitigate seasonality effect (yearly time horizon, monthly timesteps)
2. To explore the role of storage to mitigate persistent droughts (10 years time horizon)